pkg/alertmanager/alertmanager.go

```diff
@@ -369,7 +369,7 @@ func (am *Alertmanager) ApplyConfig(userID string, conf *config.Config, rawCfg s
 		return notifier
 	})
 	if err != nil {
-		return nil
+		return err
```

Comment:
`nil` looks like a mistake to me! Thanks for fixing this. LGTM for `alertmanager.go`.